parking_permits_app/models.py

```diff
@@ -0,0 +1,386 @@
+        null=False,
+    )
+    category = models.CharField(
+        _("Vehicle category"),
+        max_length=16,
+        blank=False,
+        null=False,
+        choices=CATEGORIES,
+    )
+    manufacturer = models.CharField(
+        _("Vehicle manufacturer"), max_length=32, blank=False, null=False
+    )
+    model = models.CharField(_("Vehicle model"), max_length=32, blank=False, null=False)
+    production_year = models.IntegerField(
+        _("Vehicle production_year"), blank=False, null=False
+    )
+    registration_number = models.CharField(
+        _("Vehicle registration number"), max_length=24, blank=False, null=False
+    )
+    emission = models.IntegerField(_("Emission"), blank=False, null=False)
+    last_inspection_date = models.DateField(
+        _("Last inspection date"), blank=False, null=False
+    )
+    owner = models.ForeignKey(
+        Customer,
+        verbose_name=_("Owner"),
+        on_delete=models.PROTECT,
+        related_name="vehicles_owner",
+    )
+    holder = models.ForeignKey(
+        Customer,
+        verbose_name=_("Holder"),
+        on_delete=models.PROTECT,
+        related_name="vehicles_holder",
+    )
+
+    class Meta:
+        db_table = "vehicle"
+        verbose_name = _("Vehicle")
+        verbose_name_plural = _("Vehicles")
+
+    def __str__(self):
+        return "%s - %s, %s, %s, %s" % (
+            self.id,
+            self.type,
+            self.category,
+            self.manufacturer,
+            self.model,
+        )
+
+
+class DrivingClass(models.Model):
+    id = models.UUIDField(
+        primary_key=True, unique=True, editable=False, default=uuid.uuid4
+    )
+    class_name = models.CharField(
+        _("Driving class name"), max_length=32, blank=False, null=False
+    )
+    identifier = models.CharField(
+        _("Driving class identifier"), max_length=32, blank=False, null=False
+    )
+
+    class Meta:
+        db_table = "driving_class"
+        verbose_name = _("Driving class")
+        verbose_name_plural = _("Driving classes")
+
+    def __str__(self):
+        return "%s - %s - %s" % (self.id, self.class_name, self.identifier)
+
+
+class DrivingLicence(models.Model):
+    id = models.UUIDField(
+        primary_key=True, unique=True, editable=False, default=uuid.uuid4
+    )
+    customer = models.ForeignKey(
+        Customer, verbose_name=_("Customer"), on_delete=models.PROTECT
+    )
+    driving_classes = models.ManyToManyField(DrivingClass)
+    valid_start = models.DateTimeField(_("Valid start"))
+    valid_end = models.DateTimeField(_("Valid end"))
+    active = models.BooleanField(null=False, default=True)
+
+    class Meta:
+        db_table = "driving_licence"
+        verbose_name = _("Driving licence")
+        verbose_name_plural = _("Driving licences")
+
+    def __str__(self):
+        return "%s - %s, active: %s" % (self.id, self.customer, self.active)
+
+
+class ContractType(models.Model):
+    CONTRACT_TYPES = (
```

Comment:
May I suggest to also move this `CONTRACT_TYPES` to `constants.py` so that both `ContractType` and `Product` can import and use it?

Reply:
Same here.